documentation/library_guide/parallel_api/pass_data_algorithms.rst

```diff
@@ -1,20 +1,36 @@
 Pass Data to Algorithms
 #######################
 
-When using the C++ standard execution policies, oneDPL supports data being passed to its algorithms as specified
-in the ISO/IEC 14882:2017 standard (commonly called C++17). According to the standard, the calling code
-must prevent data races when using algorithms with parallel execution policies.
+When using the C++ standard aligned (or *host*) execution policies, oneDPL supports data being passed to its
```

Comment:
```suggestion
When using the C++ standard aligned (or *host*) execution policies, |onedpl_short| supports data being passed to its
```

Reply:
This isn't needed to merge, but if there are other changes prior to final approval this would make the use consistent.